mithril-relay/src/repeater.rs

```diff
@@ -0,0 +1,128 @@
+use anyhow::anyhow;
+use mithril_common::StdResult;
+use slog_scope::debug;
+use std::{fmt::Debug, sync::Arc, time::Duration};
+use tokio::{
+    sync::{mpsc::UnboundedSender, Mutex},
+    time::Instant,
+};
+
+/// A message repeater will send a copy of the message to a channel at a given frequency
+pub struct MessageRepeater<M: Clone + Debug + Sync + Send + 'static> {
```

Comment:
Given the size of `RegisterSignerMessage`, could it be a problem to clone the message every time?

Reply:
If you mean cloning the message sent to the channel, I guess that is the expected way of passing messages to a channel.
Also, I don't really foresee any problem with the size as it's only few KB.

Can you further elaborate your concerns?